Userland/Demos/Mandelbrot/Mandelbrot.cpp

```diff
@@ -106,8 +106,11 @@ class MandelbrotSet {
         if (iteration == max_iterations)
             return iteration;
 
-        auto lz = sqrt(x * x + y * y) / 2;
```

Comment:
On the commit message:
I would suggest you to read the Contribution guidelines again, but in a nutshell, use the commit title to describe what you've done and not "Fix/Correct xxx". So prefer something like `Use log(x) instead of sqrt(x) to smooth color gradients`.